deps/install.jl

```diff
@@ -1,10 +1,13 @@
 const home = "https://github.com/JuliaSmoothOptimizers"
 const deps = Dict{AbstractString, AbstractString}(
-               "NLPModels" => "develop",
-               "OptimizationProblems" => "master",
-               "Krylov" => "develop",
-               "AmplNLReader" => "develop",
-               "Profiles" => "master")
+              "NLPModels" => "develop",
+              "OptimizationProblems" => "master",
+              "Krylov" => "develop",
+              "AmplNLReader" => "develop",
+              "Profiles" => "master")
+
+const unix_deps = Dict{AbstractString, AbstractString}(
+              "CUTEst" => "develop")
```

Comment:
This requires [gfortran](https://github.com/JuliaSmoothOptimizers/CUTEst.jl/blob/develop/.travis.yml#L14).